<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>296</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Add</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>95</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>125</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>155</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone #</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>185</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Birthdate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject #</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="add_birthdate_dateEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>180</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>65</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ethnic ID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>95</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Race</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>125</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>155</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Consent</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>185</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Audiogram</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="add_contact_add_checkBox">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>210</y>
       <width>196</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Consent to be contacted?</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="add_audiogram_dateEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>180</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDateEdit" name="add_consent_dateEdit">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="add_name_first_lineEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>60</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="add_name_last_lineEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>90</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="add_email_lineEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>120</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="add_phone_lineEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>150</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="add_subject_lineEdit">
     <property name="geometry">
      <rect>
       <x>85</x>
       <y>15</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="add_ethnic_id_comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>60</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="add_race_comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="add_gender_comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>120</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="add_pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>240</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>15</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
    <widget class="QComboBox" name="add_protocol_comboBox">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>140</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="add_age_label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>215</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Age</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Edit</string>
    </attribute>
    <widget class="QComboBox" name="edit_uid_comboBox">
     <property name="geometry">
      <rect>
       <x>305</x>
       <y>10</y>
       <width>121</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_ieee_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>45</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="cuny_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>75</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_hint_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>105</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_notes_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>165</y>
       <width>200</width>
       <height>52</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_cid_lineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>135</y>
       <width>200</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>140</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CID</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>50</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IEEE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>80</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CUNY</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>110</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>HINT</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>170</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Notes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="edit_contact_checkBox">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>220</y>
       <width>221</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Consent to be contacted?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_pushButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>240</y>
       <width>81</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_search_lineEdit">
     <property name="geometry">
      <rect>
       <x>79</x>
       <y>10</y>
       <width>126</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>15</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Search:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>15</y>
       <width>80</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Subject #</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>216</width>
       <height>126</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="edit_name_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>196</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="edit_email_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>196</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="edit_phone_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>196</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
     <widget class="QLabel" name="edit_protocol_label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>196</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Protocol</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Browse</string>
    </attribute>
    <widget class="QColumnView" name="columnView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>25</y>
       <width>471</width>
       <height>226</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Reports</string>
    </attribute>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>126</width>
       <height>251</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Admin</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="close_pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>295</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="filePath_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>300</y>
     <width>376</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>File Path</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>467</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
